digdag-server/src/main/java/io/digdag/server/rs/SessionResource.java

```diff
@@ -64,17 +65,20 @@ public SessionResource(
     @Path("/api/sessions")
     public RestSessionCollection getSessions(
             @QueryParam("last_id") Long lastId,
-            @QueryParam("page_size") Integer pageSize)
+            @QueryParam("page_size") Integer pageSize,
+            @QueryParam("page_number") Integer pageNumber)
```

Comment:
I noticed if page_number <=0, API become error in SQL execution.
I guess it is better to validate before execute query.
But page_size is also same behavior.
It is not mandatory and another PR which fix both page_size and page_number may be better.

Reply:
OK, I will fix the problem on another PR.